File: programmers/level2/python/JadenCase_문자열_만들기.py
from typing import List

def solution(s):
    ans : str = ""
    splitted : List[str] = s.split(" ")
    for string in splitted:
        if len(string) == 0:
            ans += " "
        else :
            ans += string[0].upper()
            ans += string[1:].lower()
            ans += " "    
    
    return ans[0:len(s)]
    

File: programmers/level2/go/JadenCase_문자열_만들기.go
package main

import "strings"

func solution(s string) string {
	var sb strings.Builder
	splitted := strings.Split(s, " ")
	for _, str := range splitted {
		if len(str) == 0 {
			sb.WriteString(" ")
		} else {
			sb.WriteString(strings.ToUpper(str[0:1]))
			sb.WriteString(strings.ToLower(str[1:len(str)]))
			sb.WriteString(" ")
		}
	}
	return sb.String()[0:len(s)]
}
